server/shared/auth/authenticator.js

```diff
@@ -23,6 +23,7 @@ class Auth extends Authenticator {
   }
 
   authenticate(strategy, ...args) {
+    args[0] = { session: false, ...args[0] };
```

Comment:
This is not the correct place to set the `session` option, as there is no guarantee that `args[0]` represents options at this point. Once `parseAuthenticateOptions` is executed and options are assigned, we can set the `session` option.
Also, I would like to add a comment on why we are setting the `session` option, e.g. "The passport started to be explicit about the session from v0.6.0, and since most of our authentication strategies are stateless, we are opting to avoid session by default".

Reply:
Moved setting session to false from args to options